AirCasting/SearchAndFollow/CompleteScreen/SearchSessionStreamsDownstream.swift

```diff
@@ -0,0 +1,16 @@
+// Created by Lunar on 23/03/2022.
+//
+
+import Foundation
+
+protocol SearchSessionStreamsDownstream {
+    func downloadSession(uuid: SessionUUID, completion: @escaping (Result<SearchSession, Error>) -> Void)
+}
+
+class SearchSessionStreamsDownstreamMock: SearchSessionStreamsDownstream {
+    func downloadSession(uuid: SessionUUID, completion: @escaping (Result<SearchSession, Error>) -> Void) {
+        DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(5)) {
+            completion(.success(.mock))
+        }
```

Comment:
This part is crashing on the simulator of iPhone SE 3rd gen. On the simulator of iPhone 12 everything is working fine, as well as on the physical device. I am leaving it up to you, if it is worth to make the effort to fix it now or just after you will be working with real data